BookingsAPI/Bookings.API/Controllers/PersonsController.cs

```diff
@@ -132,22 +132,18 @@ public async Task<IActionResult> GetPersonSuitabilityAnswers(string username)
 
         private PersonSuitabilityAnswerResponse BuildResponse(VideoHearing hearing, string username)
         {
-            PersonSuitabilityAnswerResponse personSuitabilityAnswer = null;
-            var participant = hearing.Participants.FirstOrDefault(p => p.Person.Username.ToLower() == username.ToLower().Trim());
-            if (participant != null && participant.SuitabilityAnswers.Any())
+            var participant = hearing.Participants.FirstOrDefault(p => p.Person.Username.Trim().ToLower() == username.Trim().ToLower());
```

Comment:
We can remove the `trim` from `p.Person.Username.Trim()` because if we get into a situation where we have an untrimmed in the db we've really messed something up and we want to find it.

Reply:
Done